<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>661</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>6</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gameInfo">
     <item row="2" column="0">
      <widget class="QPushButton" name="lopetaPeliButton">
       <property name="text">
        <string>Lopeta peli</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="uusiPeliButton">
       <property name="text">
        <string>Uusi peli</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="pelaajienPisteet"/>
   </widget>
   <widget class="QLabel" name="pelaajatLabel">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>341</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pelaajat:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pisteetLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pisteet:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vuoroLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kenen vuoro??</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vaihdaVaiheButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Seuraava vaihe</string>
    </property>
   </widget>
   <widget class="QLabel" name="mitaTehda">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridSpenWheel"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
